### PR TITLE
#8336 - triggers description

### PR DESCRIPTION
**Pull request by Vegebutcher**

### CODE REVIEW

File: G1ANT.Addon.Core/Triggers/DESCRIPTION.md [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
+3. Expand the `Triggers` entry by clicking its arrow on the left. You should see a new entry with the default `trigger1` name. Expand it.
+4. Click the field next to the `TaskName` entry and enter the full path to the robot script file you have just saved. This is the script to be run by the trigger.
+5. Click the field next to the `Class` entry and type the desired trigger name: `FileTrigger`, `MailTrigger` or `ScheduleTrigger`.
+6. Right-click the `Arguments` entry and select `Add` from the context menu.
+7. Expand the resulting `argument1` entry, click the field next to the `Key` entry and type the name of a key.
+8. Click the field next to the `Value` entry and type the key value.
+9. Repeat steps 6-8 to add more arguments.
 
-**Example**
+Another way of defining a trigger is to paste some XML code between `<Triggers>` … `</Triggers>` tags in G1ANT.Robot.config file (it’s location is provided in the *File name:* drop down list at the top of Settings window). Here’s a sample code:
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.Core/Triggers/DESCRIPTION.md]: it's location is :p

File: G1ANT.Addon.Core/Triggers/FileTrigger.md [hunk 2]
```diff
@@ -1,42 +1,45 @@
-# File Trigger
+# FileTrigger
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.Core/Triggers/FileTrigger.md]: revert it because it should be the same as Mail Structure for example

**Review by WiktoriaPrusik — CHANGES_REQUESTED** [review]

File: G1ANT.Addon.Core/Triggers/DESCRIPTION.md (hunk 1 above)
Comment (Vegebutcher) [reply G1ANT.Addon.Core/Triggers/DESCRIPTION.md]:
```suggestion
Another way of defining a trigger is to paste some XML code between `<Triggers>` … `</Triggers>` tags in G1ANT.Robot.config file (its location is provided in the *File name:* drop down list at the top of Settings window). Here’s a sample code:
```

File: G1ANT.Addon.Core/Triggers/DESCRIPTION.md [hunk 3]
```diff
@@ -1,37 +1,48 @@
 # Triggering
 
-Section triggering describes all the triggers which can be generated in specific circumstances (depending on the trigger) and can call user defined process.
-Each entry of this section should be defined as below.
+Triggers can execute user defined processes based on events. There are three triggers that can be used by G1ANT.Robot: [FileTrigger](FileTrigger.md), [MailTrigger](MailTrigger.md) and [ScheduleTrigger](ScheduleTrigger.md). Their names are self-explanatory, as they are activated on a specified file, mail or schedule event.
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/DESCRIPTION.md]:
```suggestion
Triggers can execute user defined processes based on events. There are three triggers that can be used by G1ANT.Robot: [File Trigger](FileTrigger.md), [Mail Trigger](MailTrigger.md) and [Schedule Trigger](ScheduleTrigger.md). Their names are self-explanatory, as they are activated on a specified file, mail or schedule event.
```

File: G1ANT.Addon.Core/Triggers/FileTrigger.md (hunk 2 above)
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/FileTrigger.md]:
```suggestion
# File Trigger
```

File: G1ANT.Addon.Core/Triggers/FileTrigger.md [hunk 4]
```diff
@@ -1,42 +1,45 @@
...
 
-File Trigger monitors files in the specified folder and provides information whether they were changed, deleted, renamed or there are some new created files.
+FileTrigger monitors files in a specified folder and provides information whether they were changed, deleted, renamed or there are some new created files.
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/FileTrigger.md]:
```suggestion
File Trigger monitors files in a specified folder and provides information whether they were changed, deleted, renamed or there are some new created files.
```

File: G1ANT.Addon.Core/Triggers/FileTrigger.md [hunk 5]
```diff
@@ -1,42 +1,45 @@
...
 | `FilePath` | Path to a file in the monitored folder |
-| `FileName` | Name of the file (with extension) |
-| `FileNameWithoutExtension` | Name of the file (without extension) |
+| `FileName` | Name of a file (with extension) |
+| `FileNameWithoutExtension` | Name of a file (without extension) |
 | `ChangeType` | Information about the type of modification that was applied to a file (created, changed, deleted or renamed) |
 | `OldFilePath` | Path to the file that has been changed |
 
-## Example of Defining a File Trigger in Settings
+## Example of Defining a FileTrigger in Settings
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/FileTrigger.md]:
```suggestion
## Example of Defining a File Trigger in Settings
```

File: G1ANT.Addon.Core/Triggers/FileTrigger.md [hunk 6]
```diff
@@ -1,42 +1,45 @@
...
+
+![FileTrigger Example](https://manual.g1ant.com/link/G1ANT.Manual/-assets/filetriggerexample.png)
+
+## Example of Defining a FileTrigger in the Config File
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/FileTrigger.md]:
```suggestion
## Example of Defining a File Trigger in the Config File
```

File: G1ANT.Addon.Core/Triggers/FileTrigger.md [hunk 7]
```diff
@@ -1,42 +1,45 @@
...
 	<Arguments>
-		<Argument Key="Directory">C:\Users\a\Documents\G1ANT.Robot</Argument>
+		<Argument Key="Directory">C:\Users\Robot\Documents\G1ANT.Robot</Argument>
 	</Arguments>
 </Trigger> 
 ```
 
 ## Example
 
+The script executed by the FileTrigger can make use of task arguments mentioned earlier. The following line included in the script will display the full path to a file, which triggered the script execution, along with the information on the type of change to this file:
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/FileTrigger.md]:
```suggestion
The script executed by the File Trigger can make use of task arguments mentioned earlier. The following line included in the script will display the full path to a file, which triggered the script execution, along with the information on the type of change to this file:
```

File: G1ANT.Addon.Core/Triggers/MailTrigger.md [hunk 8]
```diff
@@ -1,41 +1,45 @@
-# Mail Trigger
+# MailTrigger
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/MailTrigger.md]:
```suggestion
# Mail Trigger
```

File: G1ANT.Addon.Core/Triggers/MailTrigger.md [hunk 9]
```diff
@@ -1,41 +1,45 @@
...
 
-Mail Trigger provides information about incoming emails.
+MailTrigger provides information about incoming emails.
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/MailTrigger.md]:
```suggestion
Mail Trigger provides information about incoming emails.
```

File: G1ANT.Addon.Core/Triggers/MailTrigger.md [hunk 10]
```diff
@@ -1,41 +1,45 @@
...
 | `To` | Recipient's email address |
-| `HtmlBody` | Incoming mail content (html) |
-| `Uid` | A unique identifier of a mail given by a server |
+| `HtmlBody` | Incoming mail content (HTML) |
+| `Uid` | Unique identifier of a mail given by a server |
 | `Date` | Sent date |
 | `Priority` | Priority set by a sender |
 
-## Example of Defining a Mail Trigger in Settings
+## Example of Defining a MailTrigger in Settings
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/MailTrigger.md]:
```suggestion
## Example of Defining a Mail Trigger in Settings
```

File: G1ANT.Addon.Core/Triggers/MailTrigger.md [hunk 11]
```diff
@@ -1,41 +1,45 @@
...
+
+![MailTrigger in Settings](https://manual.g1ant.com/link/G1ANT.Manual/-assets/mailtriggerexample.png)
+
+## Example of Defining a MailTrigger in the Config File
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/MailTrigger.md]:
```suggestion
## Example of Defining a Mail Trigger in the Config File
```

File: G1ANT.Addon.Core/Triggers/MailTrigger.md [hunk 12]
```diff
@@ -44,9 +48,10 @@ Arguments generated while executing the script for each incoming email:
 
 ## Example
 
+The script executed by the MailTrigger can make use of task arguments mentioned earlier. The following line included in the script will display the title and the recipient’s name of an email, which triggered the script execution:
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/MailTrigger.md]:
```suggestion
The script executed by the Mail Trigger can make use of task arguments mentioned earlier. The following line included in the script will display the title and the recipient’s name of an email, which triggered the script execution:
```

File: G1ANT.Addon.Core/Triggers/ScheduleTrigger.md [hunk 13]
```diff
@@ -1,23 +1,27 @@
-# Schedule Trigger
+# ScheduleTrigger
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/ScheduleTrigger.md]:
```suggestion
# Schedule Trigger
```

File: G1ANT.Addon.Core/Triggers/ScheduleTrigger.md [hunk 14]
```diff
@@ -1,23 +1,27 @@
...
 
-Schedule trigger launches tasks at some specified time.
+ScheduleTrigger executes tasks at a specified time.
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/ScheduleTrigger.md]:
```suggestion
Schedule Trigger executes tasks at a specified time.
```

File: G1ANT.Addon.Core/Triggers/ScheduleTrigger.md [hunk 15]
```diff
@@ -1,23 +1,27 @@
...
+| `CrontabExpression` | yes |  | Time specified for the trigger to be called at. For more information about setting this crontab argument, please visit [Crontab](https://crontab.guru/) website. |
 
 ## Task Arguments
 
 | Trigger Argument | Description |
 | -------- | ---- |
 | `Time` | Amount of time it took the trigger to perform the action |
 
-## Example of Defining a Schedule Trigger in Settings
+## Example of Defining a ScheduleTrigger in Settings
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/ScheduleTrigger.md]:
```suggestion
## Example of Defining a Schedule Trigger in Settings
```

File: G1ANT.Addon.Core/Triggers/ScheduleTrigger.md [hunk 16]
```diff
@@ -1,23 +1,27 @@
...
+
+![ScheduleTrigger in Settings](https://manual.g1ant.com/link/G1ANT.Manual/-assets/scheduletriggerexample.png)
+
+## Example of Defining a ScheduleTrigger in the Config File
```
Comment (Vegebutcher) [inline G1ANT.Addon.Core/Triggers/ScheduleTrigger.md]:
```suggestion
## Example of Defining a Schedule Trigger in the Config File
```

File: G1ANT.Addon.Core/Triggers/FileTrigger.md (hunk 2 above)
Comment (Vegebutcher) [reply G1ANT.Addon.Core/Triggers/FileTrigger.md]: done

**Review by WiktoriaPrusik — APPROVED** [review]

**Pull request merged**